gateleen-core/src/main/java/org/swisspush/gateleen/core/event/EventBusHandler.java

```diff
@@ -240,9 +240,9 @@ public void install(Router router) {
     }
 
     @Override
-    public void resourceChanged(String resourceUri, String resource) {
+    public void resourceChanged(String resourceUri, Buffer resource) {
         if(configResourceUri() != null && configResourceUri().equals(resourceUri)){
-            log.info("Got notified about configuration resource update for "+resourceUri+" with new data: " + resource);
+            log.info("Got notified about configuration resource update for " + resourceUri);
```

Comment:
I suggest to use [Parameterized Logging](https://www.tutorialspoint.com/slf4j/slf4j_parameterized_logging.htm). (There are also other occurences)